### PR TITLE
DIP 24

### PR DESCRIPTION
**Pull request by cbergz**
Reducing LP Rewards by 50% through the merkle distributor

### CODE REVIEW

File: src/deploy-config/base-config.ts [hunk 1]
```diff
@@ -297,6 +297,15 @@ const config = {
   // Price derived is $1.53 using Binance.com DYDX/USDT on
   // Using market price of $1.53, rounded amount to be transferred is 4,314,000 DYDX
   DOT_FUNDING_AMOUNT_v2: '4314000000000000000000000',
+
+  // Update Merkle Distributor LP Rewards (DIP 24)
+  // Proposal to reduce the current LP rewards by 50%. Trading rewards will not be changed.
+  // Alpha parameter will remain unchanged.
+  // LP rewards being reduced by 50% from '1,150,685' to '575,343' per epoch.
+  UPDATE_MERKLE_DISTRIBUTOR_LP_REWARDS_AMOUNT_DIP24: '575343000000000000000000',
```
Comment (jiajames) [inline src/deploy-config/base-config.ts:305]: I believe this should be `575342500000000000000000` to stay in line with the snapshot vote, but please correct me i'm wrong (a bit out of the loop here)

File: src/lib/constants.ts [hunk 2]
```diff
@@ -57,3 +57,8 @@ export const DIP_22_IPFS_HASH = '0x11e2866e87f43c68118021b1efaf1053be1f3cb0d25c0
 // DIP_23_IPFS_HASH taken from the link below:
 // https://github.com/dydxfoundation/dip/blob/master/content/ipfs-dips/DIP-23-Ipfs-hashes.json
 export const DIP_23_IPFS_HASH = '0x5aca381042cb641c1000126d5a183c38b17492eb60a86910973d0c3f1e867f43';
+
+
+// DIP_24_IPFS_HASH taken from the link below:
+//
+export const DIP_24_IPFS_HASH = '0x0000000000000000000000000000000000000000000000000000000000000000';
```
Comment (jiajames) [inline src/lib/constants.ts:64]: can you add a new line here?

File: src/deploy-config/base-config.ts (hunk 1 above)
Comment (cbergz) [reply src/deploy-config/base-config.ts:305]: 575343000000000000000000 is the right amount included in the Proposed Change section of the snapshot. Max made a typo in the Summary section.

File: src/lib/constants.ts (hunk 2 above)
Comment (cbergz) [reply src/lib/constants.ts]: done!

**Review by jiajames — APPROVED** [review]

File: src/deploy-config/base-config.ts (hunk 1 above)
Comment (jiajames) [reply src/deploy-config/base-config.ts:305]: Ooh thanks, totally missed that

**Pull request merged**